### PR TITLE
Modify peerstore to scope to Peer ID not job ID (required for multi-job support)

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: core/services/offchainreporting/peerstore.go [hunk 1]
```diff
@@ -14,22 +14,23 @@ import (
...
+	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
 type (
 	P2PPeer struct {
 		ID        string
 		Addr      string
-		JobID     int32
+		PeerID    string
```
Comment (connorwstein) [inline core/services/offchainreporting/peerstore.go:25]: Is the thinking here that the same node could have multiple peerIDs? Or just data denormalization

File: core/store/migrations/migration1607954593/migrate.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package migration1607954593
+
+import "github.com/jinzhu/gorm"
+
+func Migrate(tx *gorm.DB) error {
```
Comment (connorwstein) [inline core/store/migrations/migration1607954593/migrate.go:5]: looks worth of a quick migration unit test?

File: core/services/offchainreporting/peerstore.go (hunk 1 above)
Comment (samsondav) [reply core/services/offchainreporting/peerstore.go:25]: Yes one node could have multiple peer IDs

File: core/store/migrations/migration1607954593/migrate.go (hunk 2 above)
Comment (samsondav) [reply core/store/migrations/migration1607954593/migrate.go:5]: What for?

Comment (connorwstein) [reply core/store/migrations/migration1607954593/migrate.go:5]: we don't actually test this update statement anywhere

**Comment by Loverboy912** [comment]
> _No description provided._

I don't know.... Please help me I'm invited you in my organization.... I don't know how to drive this only I know to create... And build... But to exit and marketing I don't know sooo please... Help me I'm invited all of u in my organisation

**Review by Loverboy912 — CHANGES_REQUESTED** [review]
Request changes

**Review by connorwstein — APPROVED** [review]

File: core/store/migrations/migration1607954593/migrate.go (hunk 2 above)
Comment (connorwstein) [reply core/store/migrations/migration1607954593/migrate.go:5]: (not a blocking comment)

**Pull request merged**